spi/data-plane-selector/data-plane-selector-spi/src/testFixtures/java/org/eclipse/edc/connector/dataplane/selector/spi/testfixtures/TestDataPlaneInstance.java

```diff
@@ -33,7 +33,7 @@
 
 @JsonDeserialize(builder = TestDataPlaneInstance.Builder.class)
 @JsonTypeName("dataspaceconnector:testdataplaneinstance")
-public class TestDataPlaneInstance implements DataPlaneInstance {
+public class TestDataPlaneInstance extends DataPlaneInstance {
```

Comment:
Is this needed anymore? Now `DataPlaneInstance` can be used instead.

Reply:
Yes i'm agree that it's not necessary